subsys/bluetooth/audio/ascs.c

```diff
@@ -3111,6 +3217,10 @@ int bt_ascs_init(const struct bt_bap_unicast_server_cb *cb)
 {
 	int err;
 
+	if (!ascs.registered) {
+		return -ENOTSUP;
+	}
```

Comment:
How can this happen?

Reply:
I can't, good catch.

Reply:
Simply by calling 'bt_bap_unicast_server_register_cb()' before 'bt_bap_unicast_server_register()'

Reply:
Oh, my bad. Github kinda showed this as being in the `control_point_notify` function xD Disregard this comment